utils/hv-tools/Makefile

```diff
@@ -0,0 +1,52 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=hv-tools
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/NikulovE/hv-tools/
+PKG_SOURCE_VERSION:=e1d320d028752afe7b7c3adbfc6f91b4305d50a2
+PKG_MIRROR_HASH:=4d9a5d823df86a2c3ad59201a3ca2438a9e679e384ee59119b7599df2a30d1a1
+
+PKG_MAINTAINER:=Evgeniy Nikulov <nikulov@live.ru>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+define Package/hv-tools
+  SECTION:=utils
+  CATEGORY:=Utilities
+  SUBMENU:=Virtualization
+  TITLE:=Hyper-V Guest Tools
+  DEPENDS:=+procd +libpthread @(TARGET_x86||TARGET_x86_64)
+endef
+
+define Package/hv-tools/description
+This package contains Hyper-V Guest Tools for OpenWrt, including various utilities and daemons.
+endef
+
+define Build/Compile
+	$(MAKE_VARS) $(MAKE) $(MAKE_FLAGS) all -C $(PKG_BUILD_DIR)/files/tools/hv
```

Comment:
What I think you need to somehow look at your repository and reorder the folder structure, which you have there. Not sure, why there is OpenWrt Makefile in the root repository, where there should be the Makefile from /files/tools/hv. Then you would not need to have this after all.

Reply:
I want to keep this structure in respect to Linux kernel source structure https://github.com/torvalds/linux/blob/master/tools/hv/

Reply:
Even in the Linux kernel source structure it is what I said. In the root directory, there is Makefile. Not OpenWrt Makefile, so there is definitely something wrong what you did.

Reply:
I tried to use Kernels source and was not able to solve issues in step hv-tools/compile - it requires to bring all kernels sources before "compile" step, and I did not find any applicable option that I can to DEPENDS to solve it.
Ok, no problems to use 
```Makefile
$(INSTALL_BIN) $(LINUX_DIR)/tools/hv/hv_kvp_daemon $(1)/usr/sbin/hv_kvp_daemon
$(INSTALL_BIN) $(LINUX_DIR)/tools/hv/hv_vss_daemon $(1)/usr/sbin/hv_vss_daemon
(INSTALL_BIN) $(LINUX_DIR)/tools/hv/hv_fcopy_daemon $(1)/usr/sbin/hv_fcopy_daemon
```
but after that I got error (one of checks for PR):
```sh
make package/hv-tools/compile V=s
```
```sh
install -d -m0755 /home/nikulove/repos/openwrt/build_dir/target-x86_64_musl/hv-tools-1.0.0/.pkgdir/hv-tools/usr/sbin
install -m0755 /home/nikulove/repos/openwrt/build_dir/target-x86_64_musl/linux-x86_64/linux-6.6.41/tools/hv/hv_kvp_daemon /home/nikulove/repos/openwrt/build_dir/target-x86_64_musl/hv-tools-1.0.0/.pkgdir/hv-tools/usr/sbin/hv_kvp_daemon
install: cannot stat '/home/nikulove/repos/openwrt/build_dir/target-x86_64_musl/linux-x86_64/linux-6.6.41/tools/hv/hv_kvp_daemon': No such file or directory
make[2]: *** [Makefile:55: /home/nikulove/repos/openwrt/build_dir/target-x86_64_musl/hv-tools-1.0.0/.pkgdir/hv-tools.installed] Error 1
make[2]: Leaving directory '/home/nikulove/repos/openwrt/feeds/packages/utils/hv-tools'
time: package/feeds/packages/hv-tools/compile#1.00#0.27#1.55
    ERROR: package/feeds/packages/hv-tools failed to build.
make[1]: *** [package/Makefile:173: package/feeds/packages/hv-tools/compile] Error 1
```
Need to compile whole firmware for x86_64 and after that this check will passed successfully

Reply:
concept is in progress

Reply:
@BKPepe do you know way how to set up dependencies to bring /hv files from Linux kernel before /hv-tools/compile operation?